src/explore.sh

```diff
@@ -14,7 +14,7 @@ function explore_main()
 {
   local ret
 
-  if [[ "$1" =~ -h|--help ]]; then
+  if [[ "$1" =~ --help ]]; then
```

Comment:
-h is for a short help, and --help is for open the man page.

Reply:
What about "-H" for header "-h" for help? :D